GeoFireTests/GFRealDataTest.m

```diff
@@ -10,16 +10,35 @@
 
 #import "TestHelpers.h"
 
+@import Firebase;
+
 @implementation GFRealDataTest
 
 - (void)setUp
 {
     [super setUp];
     NSString *randomFirebaseURL = [NSString stringWithFormat:@"https://%@.firebaseio-demo.com",
```

Comment:
Hrm, this may present a problem in the future if demo no longer exists. For now I think we'll be fine, but we may have to change this in the future.